Missing include leads to compilation error

Type `field_type` and `option` are used [here](https://github.com/NilFoundation/crypto3-algebra-marshalling/blob/master/include/nil/crypto3/marshalling/algebra/inference.hpp#L67) and [here](https://github.com/NilFoundation/crypto3-algebra-marshalling/blob/master/include/nil/crypto3/marshalling/algebra/inference.hpp#L64), but not included.

As a result, if you include this file:

```cpp
#include "nil/crypto3/marshalling/algebra/inference.hpp"
```

you will get compilation errors:

```
.../include/nil/crypto3/marshalling/algebra/inference.hpp:64:40: error: 'option' does not name a type
   64 |             using default_endianness = option::big_endian;
      |                                        ^~~~~~
.../include/nil/crypto3/marshalling/algebra/inference.hpp:67:83: error: 'field_type' was not declared in this scope
   67 |             using type = typename nil::crypto3::marshalling::types::curve_element<field_type<TEndian>,
      |                                                                                   ^~~~~~~~~~
```

Probably this includes are missing:

```cpp
#include <nil/marshalling/options.hpp>
#include <nil/marshalling/field_type.hpp>
```

--- a/include/nil/crypto3/marshalling/algebra/inference.hpp
+++ b/include/nil/crypto3/marshalling/algebra/inference.hpp
@@ -31,6 +31,9 @@
 
 #include <nil/crypto3/algebra/type_traits.hpp>
 
+#include <nil/marshalling/field_type.hpp>
+#include <nil/marshalling/options.hpp>
+
 namespace nil {
     namespace crypto3 {
         namespace marshalling {

